github-for-jira.sd.yml

```diff
@@ -204,6 +204,22 @@ alarms:
     HighSeverityAlarmWhenTooManyELB5xxErrors: null
     LowSeverityAlarmWhenTooManyBackend5xxErrors: null
     LowSeverityAlarmWhenTooManyELB5xxErrors: null
+  eventsQueueSize:
```

Comment:
The name of the alert need to be changed `eventsQueueSize:` because we don't have events queue.